far/FarEng.hlf.m4

```diff
@@ -6179,144 +6179,167 @@ copied.
 
 @Interface.CursorSize
 $ #far:config Interface.CursorSizeX#
- Параметры "Interface.CursorSize1" и "Interface.CursorSize2" позволяют задавать размер курсора в оконном и полноэкранном режимах Far Manager для режима вставки.
+ These numeric parameters control cursor size in Far console window.
+Cursor size can be set separately for insert and override mode, as well
+as for windowed and fullscreen mode.
 
- Параметры "Interface.CursorSize3" и "Interface.CursorSize4" позволяют задавать размер курсора в оконном и полноэкранном режимах Far Manager для режима замены.
-
- Значения параметров: число между 1 и 100 - процент символьной ячейки, который заполняется курсором.
-Курсор изменяется от полностью заполненной ячейки до горизонтальной строки внизу ячейки.
+@-
+ ┌──────────╥───────────────────────┬───────────────────────┐
+ │ Mode     ║ Windowed              │ Fullscreen            │           
+ ╞══════════╬═══════════════════════╪═══════════════════════╡
+ │ Insert   ║ Interface.CursorSize1 │ Interface.CursorSize2 │
+ ├──────────╫───────────────────────┼───────────────────────┤
+ │ Override ║ Interface.CursorSize3 │ Interface.CursorSize4 │
+ └──────────╨───────────────────────┴───────────────────────┘
+@+
 
- Значения параметров равные нулю позволяют использовать системные настройки консоли.
+ The parameters specify the fraction of the character cell in percents
+filled by the cursor. Parameters’ values may vary from 1 to 100
+corresponding to the cursor changing from the single horizontal line
+at the bottom of the cell to the solid block filling the entire cell.
+If parameter’s value is zero, the system console setting is used.
 
- По умолчанию:
- Interface.CursorSize1 = 15
- Interface.CursorSize2 = 10
- Interface.CursorSize3 = 99
- Interface.CursorSize4 = 99
+ Default values:
+ Interface.CursorSize1: 15
+ Interface.CursorSize2: 10
+ Interface.CursorSize3: 99
+ Interface.CursorSize4: 99
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
 @System.WordDiv
 $ #far:config System.WordDiv#
- Параметр "System.WordDiv" позволяет задавать дополнительные (кроме пробела и табуляции) cимволы-разделители слов.
+ This string parameter defines additional word delimiters besides
+#Space# (U+0020) and #Tab# (U+0009).
```

Comment:
Do we really need to mention the corresponding unicode code points for space and tab?

Reply:
Probably not. Removed everywhere. Thanks.